packages/frontend-shared/src/locales/en-US.json

```diff
@@ -120,11 +120,10 @@
     "newSpecButton": "New Spec",
     "searchPlaceholder": "Search Specs",
     "componentSpecsHeader": "Component Specs",
-    "e2eSpecsHeader": "E2E Specs",
-    "gitStatusHeader": "Git Status",
+    "e2eSpecsHeader": "E2E specs",
```

Comment:
Are we intentionally capitalizing the leading word only? Figma still seems to show the previous capitalization, but I might have missed an update somewhere.

Reply:
Yeah, we're meant to be using sentence case although Figma has a mix of sentence and title case. There's a JIRA card for it: https://cypress-io.atlassian.net/browse/UNIFY-1635

Reply:
Ah, I see. Thanks @rachelruderman!